// Distributed under the MIT software license, see the accompanying
// file LICENSE or http://www.opensource.org/licenses/mit-license.php.

syntax = "proto3";

package qrl;

service Base {
  rpc GetNodeInfo (GetNodeInfoReq) returns (GetNodeInfoResp) {}
}

message GetNodeInfoReq
{
}

message GetNodeInfoResp
{
    string version = 1;
    string grpcProto = 2;
}
